app/services/bulk_suite_selector.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+#
+# Service object to handle the bulk assignment of suites to groups. Ensures that
+# suites are available, groups don't have suites assigned, and optionally that
+# suites belong to the appropriate draw.
+class BulkSuiteSelector
```

Comment:
You're right, it basically is a form object. I'll rename / move it as appropriate.